internal/reconcile/atomic_release.go

```diff
@@ -290,6 +297,11 @@ func (r *AtomicRelease) actionForState(ctx context.Context, req *Request, state
 		//  field, but should be removed in a future release.
 		req.Object.Status.LastAppliedRevision = req.Object.Status.History.Latest().ChartVersion
 
+		if forceRequested {
+			log.Info(msgWithReason("forcing upgrade for in-sync release", "force requested through annotation"))
+			return NewUpgrade(r.configFactory, r.eventRecorder), nil
+		}
```

Comment:
Same question around user desire here: do they want to force an upgrade, or do they only want to force it forward when it is in some failed state?

Reply:
After thinking about this some more, and discussing it with @darkowlzz, I think the current behavior is what should be expected.

Given if someone wants to continue from the point where it runs into "retries exhausted", they can simply use the reset annotations. If force would do the same, it would bear no value to someone who has "unlimited retries" (`retries: -1`) configured, and this feature would be void.